pygfx/materials/_base.py

```diff
@@ -184,6 +191,18 @@ def depth_test(self, value):
             raise TypeError("Material.depth_test must be bool.")
         self._store.depth_test = bool(value)
 
+    @property
+    def write_pick(self):
+        """Whether this material is picked by the pointer."""
+        return self._store.write_pick
```

Comment:
Reading it like this I think perhaps this prop should be named `pickable` or something similar. The fact that *something* has to be written to make it so, is a fact that we don't need to concern the user with. @Korijn what do you think?

Reply:
As I proposed earlier I prefer the property to be named `pick_write` so that it will be consistent with other similar planned properties `depth_write` and `depth_test`. It's also more discoverable (with things like search and autocomplete)

Reply:
Ok. I noticed that difference between your two comments. I meant to bring it up earlier. Sorry for not flagging.
I’ll make the change to the property name.